CommonVoice-Data/requirements.txt

```diff
@@ -5,7 +5,7 @@ chardet==3.0.4
 decorator==4.3.0
 EbookLib==0.17.1
 future==0.16.0
-Gutenberg==0.6.1
+#Gutenberg==0.6.1
```

Comment:
Heu, c'est quoi ça ?

Reply:
j'ai désactivé cette ligne car elle porte des conflits dans l'environnement

Reply:
> j'ai désactivé cette ligne car elle porte des conflits dans l'environnement

Sauf que c'est utile pour l'extraction d'autres sources de données. Quels conflits ça génère ?

Reply:
lorsque je fais : pip install -r requirements.txt

le script s'arrête à cet endroit et affiche ceci : 

Collecting bsddb3>=6.1.0 (from Gutenberg==0.6.1->-r requirements.txt (line 8))
  Using cached https://files.pythonhosted.org/packages/e9/fc/ebfbd4de236b493f9ece156f816c21df0ae87ccc22604c5f9b664efef1b9/bsddb3-6.2.6.tar.gz
    Complete output from command python setup.py egg_info:
    Can't find a local Berkeley DB installation.
    (suggestion: try the --berkeley-db=/path/to/bsddb option)

Reply:
C'est pas une incompatibilité, c'est parce que tu n'as pas configuré le support BerkeleyDB ...

Reply:
C'est bon, j'ai corrigé de mon coté